vagrant/ansible/roles/samba.setup/templates/cephfs/smb_share.conf.j2

```diff
@@ -0,0 +1,7 @@
+[{{ samba_share }}]
+comment = For samba share of volume {{ cluster_volume }}
+vfs objects = acl_xattr
```

Comment:
Did we purposefully left out _vfs_cephfs_ i.e `vfs objects = acl_xattr cephfs`?

Reply:
Yes, I'll add support for VFS on a future PR.